src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -118,9 +124,64 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         this.exercise.participationStatus = participationStatus(this.exercise);
         this.isAfterAssessmentDueDate = !this.exercise.assessmentDueDate || moment().isAfter(this.exercise.assessmentDueDate);
         this.exerciseCategories = this.exerciseService.convertExerciseCategoriesFromServer(this.exercise);
+        if (this.exercise.type === ExerciseType.PROGRAMMING) {
+            this.getProgrammingExercise();
+        }
         this.subscribeForNewResults();
     }
 
+    /**
```

Comment:
if you cannot prevent to add code into "production" files, then please move them into a common section in the source code and mark it with something like

// ################## ONLY FOR LOCAL TESTING PURPOSE -- START ##################

here comes the code

// ################## ONLY FOR LOCAL TESTING PURPOSE -- END ##################

Reply:
i added these kind of comments